src/store/Account.js

```diff
@@ -80,6 +80,16 @@ const getters = {
       .filter(x => x.status === 'waiting')
   },
 
+  // todo: 'you' -> accountId :)
```

Comment:
👍

Reply:
We will change it as soon as Iroha is ready.